index.js

```diff
@@ -46,6 +46,11 @@ bot.on('ready', () => {
             message: `Battery status: ${stdout}`,
           });
         });
+      } else if (message.match(/saren/i)) {
```

Comment:
I don't know if this should be a else if clause, since this may occur in tandem with other functionality. For example, we still want to enforce r9k while doing this.